extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-thrift-server-chart/README.md

```diff
@@ -48,6 +49,9 @@ _**NOTE:**_ *the version should match the aiSSEMBLE project version.*
 If you are migrating from the v1 version of the spark-infrastructure chart, use the tables below to apply any
 existing customizations from the old chart to the new v2 chart.
 
+Any connections to the thrift server will need to be updated to include the new `.Values.app.name`. For instance the 
```

Comment:
Do we want to include the old thrift server url so it might be easier to find in a downstream?

Reply:
Yeah, I can add that in. It was originally defaulted to `spark-infrastructure`.